sdk/keyvault/keyvault-keys/test/CRUD.test.ts

```diff
@@ -39,7 +39,12 @@ describe("Keys client - create, read, update and delete operations", () => {
     await testClient.flushKey(keyName);
   });
 
+  // If this test is not skipped in the browser's playback, no other test will be played back.
+  // This is a bug related to the browser features of the recorder.
```

Comment:
Do we want to log an issue for tracking the long term fix? Or is there no long term fix needed?

Reply:
I added this as the first item on this issue: https://github.com/Azure/azure-sdk-for-js/issues/4229